docs/pages/deploy-a-cluster/teleport-enterprise/license.mdx

```diff
@@ -1,18 +1,41 @@
 ---
 title: Enterprise License File
-description: Teleport enterprise license file configuration parameters and requirements
+description: How to manage your Teleport Enterprise license file.
 ---
 
-This guide explains the use of a license file with Teleport Enterprise.
```

Comment:
Should we add a warning at the top when `scope==cloud` to say that cloud licenses are managed for you and this guide does not apply to cloud?